Makefile

```diff
@@ -0,0 +1,97 @@
+# Makefile for working with a Python module
+
+##########################################################################
+## REQUIREMENTS
+#
+# The following packages are required to run this makefile:
+#   pytest 				For running tests
+#   coverage 			For running test coverage
+#   cloc 				For counting code
```

Comment:
Should shell/bash installation be specified? `cloc` can be installed via pip, but it required a system/os package installation for me.

Reply:
Ahh, yes, very good point -  it's worth pointing out that most of these are Python packages, where as cloc is a command line tool (it's not the same tool as the `cloc` that is on PYPI I think). I've split out the requirements, to point out what kind of thing cloc is.